hddm/models/hddm_info.py

```diff
@@ -126,6 +126,8 @@ def _create_stochastic_knodes_info(self, include):
             knodes.update(self._create_family_normal_normal_hnormal('v', value=2, g_mu=2, g_tau=3**-2, std_std=2))
         if 't' in include:
             knodes.update(self._create_family_gamma_gamma_hnormal('t', g_mean=.4, g_std=0.2, value=0.001, std_std=1, std_value=0.2))
+        if 'alpha' in include:
```

Comment:
This should also be pushed into your own class.

Reply:
I haven't found a good way to this, while still being able to regress against the new parameter. The problem is that if I update the knodes with alpha after the following line it doesn't create alpha_Intercept:

        # Create all stochastic knodes except for the ones that we want to replace
        # with regressors.
        knodes = super(HDDMrlRegressor, self)._create_stochastic_knodes(include.difference(self.reg_outcomes))

Any ideas how to add alpha inside my own class but before creating knodes?

Reply:
in your overloaded method you should be able to just do:
```
knodes = super(HDDMrlRegressor, self)._create_stochastic_knodes(includes)
if 'alpha' in include:            
    knodes.update(self._create_family_normal_normal_hnormal('alpha', value=0, g_tau=50**-2, std_std=10))
return knodes
```

Reply:
Sorry, there's probably something trivial about the overloaded method I'm not getting. I've tried adding the knode for alpha at line 242 in HDDMrlRegressor.py, but then it does not create alpha_Intercept if (alpha~BOLD). Where do you suggest I add this?

knodes = super(HDDMrlRegressor, self)._create_stochastic_knodes(includes)
if 'alpha' in include:            
    knodes.update(self._create_family_normal_normal_hnormal('alpha', value=0, g_tau=50**-2, std_std=10))
return knodes